apps/omg/lib/omg/state/transaction.ex

```diff
@@ -143,13 +143,13 @@ defprotocol OMG.State.Transaction.Protocol do
   @doc """
```

Comment:
is there a more simple approach to implement what this protocol is trying to achieve?